pkg/gatewayserver/io/ws/lbslns/timesync.go

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lbslns
```

Comment:
I prefer this to be a part of the `downstream` file since this is just a downstream message.

Reply:
As the logic grew a bit https://github.com/TheThingsNetwork/lorawan-stack/pull/4805/commits/0ab1b1374f7654de0b8da269cc204607b1dc12cd, I'm thinking about keeping this as a separate file. If we discard that commit, I can move the singular function to `downstream.go`.